### ISSUE TITLE
Fork adding support for ES modules

### ISSUE THREAD

**Issue by drjeffjackson** [expressjs/generator#315]
I have made a [fork](https://github.com/drjeffjackson/expressjs-generator/tree/esm) that adds an --esm switch that generates an ES modules version of the app.  Are you accepting pull requests, and if so, would you be interested in this one?

**Comment by drjeffjackson** [comment]
Thanks, @UlisesGascon! PR submitted.

**Comment by joeyguerra** [comment]
PR #314 adds Python 2 to the runner, which should fix some of the failing tests in #316.

**Comment by drjeffjackson** [comment]
> How do I make sure I'm installing your fork and not the original? Both have the same package name.

My fork is in pull request #316.  It is still waiting to be approved for merging, but you're welcome to clone it from https://github.com/drjeffjackson/expressjs-generator/tree/esm. 
> 
> EDIT: The README of your fork is literally copy-paste from this repo's README.

The README of the esm branch of my clone of the repo has a slightly modified README that adds documentation of a new --es6 switch.

**Comment by IgorArnaut** [comment]
> > How do I make sure I'm installing your fork and not the original? Both have the same package name.
> 
> My fork is in pull request [#316](https://github.com/expressjs/generator/pull/316). It is still waiting to be approved for merging, but you're welcome to clone it from https://github.com/drjeffjackson/expressjs-generator/tree/esm.
> 
> > EDIT: The README of your fork is literally copy-paste from this repo's README.
> 
> The README of the esm branch of my clone of the repo has a slightly modified README that adds documentation of a new --es6 switch.

Thank you for clarifying, but I don't believe that it will be merged. There was a similar six year old PR that was not merged. https://github.com/expressjs/generator/pull/200

**Comment by drjeffjackson** [comment]
> Thank you for clarifying, but I don't believe that it will be merged. There was a similar six year old PR that was not merged. [#200](https://github.com/expressjs/generator/pull/200)

I've just made a change requested by @ljharb (removed `node='module'` from package.json).  Perhaps that will make a difference.

**Comment by clwy-cn** [comment]
Since there hasn't been a merge for a long time, I've created a new npm package based on this. Please try it here:
https://github.com/clwy-cn/clwy-express-generator

## Features

- **🚀 ES6 Support**: Modern and cleaner code.
- **🔀 Split Routes**: Separate files for easier management and maintenance.
- **📁 Modular Middleware**: Dedicated folder for middleware.
- **🔧 Environment Variables**: Easier multi-environment configuration.
- **🔄 Nodemon Integration**: Auto-restart during development.
- **📄 Added basic README.md**: Describes the project as a simple Express app with static file serving and basic routing.
- **📦 Built-in Configs**: Includes `.prettierrc` (code formatting) and `.gitignore` (file exclusion) by default.

**Comment by tomasantunes** [comment]
I agree with this change. I'm trying to create extra files and it's a pain.
